src/Microsoft.AspNet.Authentication/ClaimsTransformationAuthenticationHandler.cs

```diff
@@ -0,0 +1,106 @@
+// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
```

Comment:
The end user doesn't really need to understand the implementation. The usage is to simply call configureClaimsTranformation to plug in their xform with the understanding that it will be invoked whenever authenticate is called. And once on HttpContext.User.